core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorSpec.scala

```diff
@@ -477,6 +478,43 @@ class SubmissionMonitorSpec(_system: ActorSystem) extends TestKit(_system) with
     }
   }
 
+  it should "attach outputs and not deadlock with multiple submissions all updating the same entity at once" in withDefaultTestDatabase { dataSource: SlickDataSource =>
+    //create a bunch of submissions all running on the same two entities
+    val numSubmissions = 50
```

Comment:
I used `withCustomTestDatabase` recently to do something similar:

https://github.com/broadinstitute/rawls/blob/develop/core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/SubmissionApiServiceSpec.scala#L57

Might be cleaner?